data/seriesRepository.go

```diff
@@ -14,29 +15,58 @@ type SeriesRepository struct {
 type transformation struct {
 	Statement        string
 	PlaceholderCount int
+	Label            string
 }
 
 const (
-	Levels = "lvl"
-	YoyPCh = "pc1"
-	YTD    = "ytd"
+	Levels           = "lvl"
+	YOYPercentChange = "pc1"
+	YTDPercentChange = "ytdpc1"
+	YOYChange        = "ch1"
+	YTDChange        = "ytdch1"
 )
 
 var transformations map[string]transformation = map[string]transformation{
-	"lvl": {
+	Levels: { // untransformed value
 		Statement:        `SELECT date, value FROM data_points WHERE series_id = ? and current = 1;`,
 		PlaceholderCount: 1,
+		Label: "lvl",
 	},
-	"pc1": {
+	YOYPercentChange: { // percent change from 1 year ago
 		Statement: `SELECT t1.date, (t1.value/t2.last_value - 1)*100 AS yoy
 				FROM (SELECT value, date, DATE_SUB(date, INTERVAL 1 YEAR) AS last_year
 				FROM data_points WHERE series_id = ? AND current = 1) AS t1
 				LEFT JOIN (SELECT value AS last_value, date
 				FROM data_points WHERE series_id = ? and current = 1) AS t2
 				ON (t1.last_year = t2.date);`,
 		PlaceholderCount: 2,
+		Label: "pc1",
 	},
-	"ytd": {
+	YOYChange: { // change from 1 year ago
+		Statement: `SELECT t1.date, t1.value - t2.last_value AS yoy
+				FROM (SELECT value, date, DATE_SUB(date, INTERVAL 1 YEAR) AS last_year
+				FROM data_points WHERE series_id = ? AND current = 1) AS t1
+				LEFT JOIN (SELECT value AS last_value, date
+				FROM data_points WHERE series_id = ? and current = 1) AS t2
+				ON (t1.last_year = t2.date);`,
+		PlaceholderCount: 2,
+		Label: "pc1",
```

Comment:
I note that label "pc1" is repeated for both YOYChange and YOYPercentChange. Similarly for "ytd" below. Pointing out in case it's a bug.